### DISCUSSION TITLE
Yolov4 training

### DISCUSSION THREAD

**Discussion by nayamac** [AlexeyAB/darknet#7993] [Q&A]
@AlexeyAB I am using yolov4, my map is vey low so I used -show_imgs flag at the end of training command, but most of the images Were cropped and there was no bounding boxes in them, My question is why were the images cropped ?

**Comment by AlexeyAB** [comment]
@nayamac 

> I am using yolov4, my map is vey low

Show screenshot of your mAP.

----

> but most of the images Were cropped

It is normal. It is just data augmentation.

----

> there was no bounding boxes in them

Labels in your dataset are wrong. Check them by using one of these utils: https://github.com/AlexeyAB/darknet#how-to-mark-bounded-boxes-of-objects-and-create-annotation-files

**Reply by nayamac** [reply]
I am using yolov4 on a custom dataset(the size of my images is 200x200), I have a really bad map and when i use -show_imgs flag at the end of training command, some of the images got mixed together, other images got cropped, so I want to know the right value of height and width to use because I used 416 and 192 but they both returned bad map.
![Capture d’écran (152)](https://user-images.githubusercontent.com/46224302/128947021-28cda9fc-cbb0-41f2-b25f-8663d6b77472.png)
and I used labelimg for labeling my images

**Reply by AlexeyAB** [reply]
Try to train 6000 iterations.

**Reply by nayamac** [reply]
When i tried 6000 iterations the results were 9%

**Reply by AlexeyAB** [reply]
> -show_imgs flag at the end of training command, but most of the images Were cropped and there was no bounding boxes in them,

If you don't see bboxes, then your labels are wrong. Try to put txt-files with labels into the same directory where are images. And re-check labels. Check that you saved them in yolo-format.

**Reply by nayamac** [reply]
I saved them in yolo format, the txt-files are in the same directory with the images, that's why i don't know what went wrong, i followed every step, but the results didn't get any better

**Reply by AlexeyAB** [reply]
1. Can you show several screenshots of images from `-show_imgs` ?
2. Can you attach the training log?

**Comment by stephanecharette** [comment]
Load your images and annotations in DarkMark.  My money is the annotations aren't right, and [DarkMark](https://www.ccoderun.ca/darkmark/Summary.html#DarkMarkEditor) will immediately show you if that is the case.

**Reply by nayamac** [reply]
Okey i will
